### PR TITLE
New Resource azurerm_stream_analytics_output_powerbi

### PR DESCRIPTION
**Pull request by jiaweitao001**
* Adding Power BI resource, to make Power BI as a stream analytics job output.

### CODE REVIEW

File: website/docs/r/stream_analytics_output_powerbi.html.markdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+* `dataset` - (Required) The name of the Power BI dataset.
+
+* `table` - (Required) The name of the Power BI table under the specified dataset.
+
+* `group_id` - (Required) The ID of the Power BI group, this must be a valid UUID.
+
+* `group_name` - (Required) The name of the Power BI group. Use this property to help remember which specific Power BI group id was used.
+
+* `authentication_mode` - (Optional) Authentication mode. Default to "Msi".
```
Comment (katbyte) [inline website/docs/r/stream_analytics_output_powerbi.html.markdown:55]: grammar and correct quoting

```suggestion
* `authentication_mode` - (Optional) Authentication mode. Defaults to `Msi`.
```

File: internal/services/streamanalytics/stream_analytics_output_powerbi_resource_test.go [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+func (r StreamAnalyticsOutputPowerBIResource) updated(data acceptance.TestData) string {
+	template := r.template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_stream_analytics_output_powerbi" "test" {
+  name                      = "acctestoutput-%d"
+  stream_analytics_job_name = azurerm_stream_analytics_job.test.name
+  resource_group_name       = azurerm_stream_analytics_job.test.resource_group_name
```
Comment (katbyte) [inline internal/services/streamanalytics/stream_analytics_output_powerbi_resource_test.go:96]: could these be combined
```suggestion
  stream_analytics_job_id = azurerm_stream_analytics_job.test.id
```

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/stream_analytics_output_powerbi.html.markdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Stream Output. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Stream Analytics Job exists. Changing this forces a new resource to be created.
+
+* `stream_analytics_job_name` - (Required) The name of the Stream Analytics Job. Changing this forces a new resource to be created.
```
Comment (katbyte) [inline website/docs/r/stream_analytics_output_powerbi.html.markdown:45]: this needs to be updated to reflect schema?

**Review by katbyte — CHANGES_REQUESTED** [review]

File: website/docs/r/stream_analytics_output_powerbi.html.markdown (hunk 3 above)
Comment (jiaweitao001) [reply website/docs/r/stream_analytics_output_powerbi.html.markdown:45]: Thanks for reminding. Will update.

**Review by katbyte — APPROVED** [review]
Thanks @jiaweitao001 ! LGTM now 🔥

**Pull request merged**